src/config/aws.ts

```diff
@@ -86,3 +79,24 @@ export function getKMSConfig(): KmsConfig {
     kmsKeyId: getKmsKeyId(),
   };
 }
+
+export function getDBConfig(config: AwsConfig = getAWSConfig()): ClientConfiguration | DynamoDBClientConfig {
+  const dbConfig:any = {};
+
+  if (config.accessKeyId || config.secretAccessKey) {
+    dbConfig.credentials = {
+      accessKeyId: config.accessKeyId || "",
```

Comment:
Why is ""  needed?  are the keys not present for localstack?

Reply:
They are - just being defensive so that if either one the values is not provided the code does not break